playwright/tests/page-objects/inscription/Table.ts

```diff
@@ -1,169 +1,174 @@
+    this.header = headerLocator.nth(0);
+  }
+
+  async fill(values: string[]) {
+    let value = 0;
+    for (let column = 0; column < this.columns.length; column++) {
+      if (this.columns[column] !== 'label') {
+        const cell = this.column(column);
+        await cell.fill(values[value++]);
+      }
+    }
+  }
+
+  column(column: number) {
+    return new Cell(this.page, this.locator, column, this.columns[column]);
+  }
+
+  async expectValues(values: string[]) {
+    let value = 0;
+    for (let column = 0; column < this.columns.length; column++) {
+      if (this.columns[column] !== 'label') {
+        const cell = this.column(column);
+        await cell.expectValue(values[value++]);
+      }
+    }
+  }
+
+  async remove(withoutHeader?: boolean) {
+    await this.locator.click();
+    await this.page.keyboard.press('Escape');
+    await this.page.getByRole('button', { name: 'Remove row' }).click();
+    if (!withoutHeader || withoutHeader === undefined) {
+      await this.header.click();
+    }
+  }
+
+  async dragTo(targetRow: Row) {
+    const source = this.locator.locator('.ui-dnd-row-handleicon');
+    const target = targetRow.locator.locator('.ui-dnd-row-handleicon');
+    await source.dragTo(target);
+  }
+}
+
+export class Cell {
+  private readonly locator: Locator;
+  private readonly textbox: Locator;
+  private readonly select: Select;
+  private readonly combobox: Combobox;
+
+  constructor(readonly page: Page, rowLocator: Locator, column: number, readonly columnType: ColumnType) {
+    this.locator = rowLocator.getByRole('cell').nth(column);
+    this.textbox = this.locator.getByRole('textbox');
+    this.select = new Select(page, this.locator);
+    this.combobox = new Combobox(page, this.locator);
+  }
+
+  async fill(value: string) {
+    switch (this.columnType) {
+      case 'label':
+        throw new Error('This column is not editable');
+      case 'text':
+        await this.fillText(value);
+        break;
+      case 'expression':
+        await this.fillExpression(value);
+        break;
+      case 'select':
+        await this.select.choose(value);
+        break;
+      case 'combobox':
+        await this.combobox.fill(value);
+        break;
+    }
+  }
+
+  async expectValue(value: string) {
+    switch (this.columnType) {
+      case 'select':
+        await this.select.expectValue(value);
+        break;
+      case 'combobox':
+        await this.combobox.expectValue(value);
+        break;
+      default:
+        await expect(this.textbox).toHaveValue(value);
+    }
+  }
+
+  async expectEmpty() {
+    await expect(this.textbox).toBeEmpty();
+  }
+
+  private async fillText(value: string) {
+    const input = this.textbox;
+    await input.fill(value);
+    await input.blur();
+  }
+
+  private async fillExpression(value: string) {
+    const code = new ScriptCell(this.page, this.textbox, this.locator);
+    await code.fill(value);
+  }
+
+  async focusScriptCell() {
```

Comment:
I only added this function, no idea, why the whole file got rewritten.

Reply:
maybe wrong line endings? maybe it had clrf before and now it is only LF or vise versa.
you can check this in if you have the file open in vscode (bottom right).
And it should be LF.
So if it changes now to LF then let it be :)

Reply:
Quite possible, I checked it, and in any case, it is now set to LF 👍